Vision/Vision.bs

```diff
@@ -85,7 +85,7 @@ Boilerplate: omit conformance
 	and its impact will continue to grow in the future, 
 	as it expands reach, knowledge, education and services even more broadly.
 	We believe the World Wide Web should be 
-	inclusive and respectful of its users: 
+	inclusive and respect the rights of all its users: 
```

Comment:
I think I want to push back a little and say I might prefer the original wording for "respectful". I took this phrasing to mean our goal is to build a web that is conscious of the needs, rights, and autonomy of users. "Respect" is a multi-faceted word. Making the change to "respect the rights of all its users" narrows that context. I think we want to respect more than just rights.